src/components/core/Alert.vue

```diff
@@ -43,43 +42,35 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import { mdiClose } from '@mdi/js'
 import { mapActions, mapState } from 'vuex'
 
+// TODO: remove later when https://github.com/vuetifyjs/vuetify/issues/11021 is fixed
+const colors = new Map([
+  ['error', 'red'],
+  ['success', 'green'],
+  ['warning', 'amber']
+])
+
 export default {
   name: 'Alert',
 
-  data () {
-    return {
-      // TODO: remove later when https://github.com/vuetifyjs/vuetify/issues/11021 is fixed
-      colors: new Map([
-        ['error', 'red'],
-        ['success', 'green'],
-        ['warning', 'amber']
-      ]),
-      svgPaths: {
-        close: mdiClose
-      },
-      snackbar: alert
-    }
-  },
-
   computed: {
     ...mapState(['alert'])
   },
 
   methods: {
     ...mapActions(['setAlert']),
     getColor (type) {
-      return this.colors.get(type) || ''
+      return colors.get(type) || ''
     },
     /**
      * Dismisses the alert from the UI, also removing it from the Vuex store.
-     *
-     * @param {Function} toggleFunction - the original Vuetify toggle function
-     * @see https://vuetifyjs.com/en/api/v-alert/
      */
     closeAlert () {
-      this.snackbar = false
       this.setAlert(null)
     }
+  },
+
+  icons: {
+    mdiClose
   }
```

Comment:
There was no need for `svgPaths` to be in `data` because it doesn't need to be reactive. I have moved it to an option and renamed, accessible via `$options.icons`

Reply:
So anything that Vue doesn't recognise as a regular config section (e.g. `data`, `computed`, `methods`, etc) is automatically made available as `$options`?